### ISSUE TITLE
Crash with the Research Table

### ISSUE THREAD

**Issue by Neodark7** [Azanor/thaumcraft#1525]
http://pastebin.com/3djygFdC

**Comment by Azanor** [comment]
Do you get this all the time, or was it a one-time thing?

**Comment by Neodark7** [comment]
Nope really random, depends of the video settings apperently but i dont think it's optifine related i had same once without optifine and now it's work with it.

**Comment by Azanor** [comment]
While I hate to do it, I think I might have to play the "mod compatibility" card. The error reported is a rendering bug that should either happen all the time or not at all. I can only suspect that something is not properly closing the tesselator which break my table rendering. This will be very hard to track down.

**Comment by Neodark7** [comment]
Yeah i can understand that, so i close the issue for now and re-open it if i get that crash again and maybe more infos ?

**Comment by Azanor** [comment]
Keep it open, but if you can post additional crash logs or get more info it would be much appreciated. :)

**Comment by Neodark7** [comment]
Sure im not on my computer right now but if i already have the yesterday log i'll post it as soon as i can. Thank you :)

**Comment by FixTheBlocks** [comment]
I'm having the same problem. Or rather, I know how to prevent the crash for myself. I have removed Waila and I could finally load into the game.

The odd part is that my world did not change. What caused the problem in the first place was me logging out in a position which resulted in me looking at the Research Table upon loading the world.

**Comment by Neodark7** [comment]
Yeah good point same for me for that "What caused the problem in the first place was me logging out in a position which resulted in me looking at the Research Table upon loading the world." 
I don't know if its waila related

**Comment by Pyr0cat** [comment]
Oh hey, I'm having this problem too except it won't let my world load at all now.
I think it is related to Waila, here's my crash report: http://pastebin.com/WHSPHEsc

**Comment by OronDF343** [comment]
Here's another crash in SMP, only crashes the client: http://pastebin.com/svNJM3GA
Probably related to logging out in a location where the Table is rendered
